src/main/java/emu/grasscutter/game/managers/InventoryManager.java

```diff
@@ -38,6 +39,8 @@ public class InventoryManager {
 	
 	private final static int RELIC_MATERIAL_1 = 105002; // Sanctifying Unction
 	private final static int RELIC_MATERIAL_2 = 105003; // Sanctifying Essence
+	private final static int RELIC_MATERIAL_EXP_1 = 2500; // Sanctifying Unction
+	private final static int RELIC_MATERIAL_EXP_2 = 10000; // Sanctifying Essence
```

Comment:
Please do not hardcode if you can find the values in the excels.

Reply:
Can't find them :(

Reply:
Its no problem then.